oaknational/Oak-Web-Application
# fix(searchbar): make searchbar white on ios

## Description

make search bar white on all ios browsers

## Issue(s)

Fixes #1340 

## How to test

1. Go to search page on ios device
2. Click on _______
3. You should see search bar background as white

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
